### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by SukhsimranpreetSekhon**

### CODE REVIEW

File: API/user/get.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const User = Models.user;
 
 const getUser = async (request, response) => {
   response.status(200).json(await User.findAll());
+  const wtf = await User.findAll();
```
Comment (Rizvida1) [inline API/user/get.js]: Is this a used variable?

File: API/user/post.js [hunk 2]
```diff
@@ -2,10 +2,11 @@ const Models = require("../../models");
 const User = Models.user;
 
 const createUser = async (request, response) => {
-  User.findOrCreate({
+  await User.findOrCreate({
     where: { email: request.body.email },
     defaults: { name: request.body.name }
   }).then(async ([user, created]) => {
+    console.log("THIS SHOULD BE NULL: ", await user.getProfile(), "The end");
```
Comment (Rizvida1) [inline API/user/post.js]: Delete

File: test/user/get.test.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      //console.log("Name: ", res.json.mock.calls[0][0][0].name);
+      expect(res.json.mock.calls[0][0][0]).toHaveProperty(
+        "email",
+        "john.doe@canada.ca"
+      );
+      //console.log("Email: ", res.json.mock.calls[0][0].email);
+      expect(res.json.mock.calls[0][0][0]).toHaveProperty("id", id);
+      //console.log("Id: ", res.json.mock.calls[0][0].id);
+      expect(res.json.mock.calls[0][0][0]).toHaveProperty("inactive", false);
+      //console.log("Inactive: ", res.json.mock.calls[0][0].inactive);
```
Comment (Rizvida1) [inline test/user/get.test.js]: Clean up comments

File: test/user/get.test.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      //console.log("Name: ", res.json.mock.calls[0][0][0].name);
+      expect(res.json.mock.calls[0][0][0]).toHaveProperty(
+        "email",
+        "john.doe@canada.ca"
+      );
+      //console.log("Email: ", res.json.mock.calls[0][0][0].email);
+      expect(res.json.mock.calls[0][0][0]).toHaveProperty("id", "012345");
+      //console.log("Id: ", res.json.mock.calls[0][0][0].id);
+      expect(res.json.mock.calls[0][0][0]).toHaveProperty("inactive", false);
+      //console.log("Inactive: ", res.json.mock.calls[0][0][0].inactive);
```
Comment (Rizvida1) [inline test/user/get.test.js]: Clean up code

File: API/admin/get.js [hunk 5]
```diff
@@ -36,10 +37,20 @@ const getFlagged = async (request, response) => {
...
+    await User.findOne({ where: { id: id } }).then(row =>
       response.status(200).json({ value: row.inactive })
     );
+
+    const print = await User.findOne({ where: { id: id } });
+
+    console.log(
+      "The End: ",
+      print.then(row => response.status(200).json({ value: row.inactive }))
+    );
```
Comment (Rizvida1) [inline API/admin/get.js]: Is this needed?

File: API/admin/get.js [hunk 6]
```diff
@@ -36,10 +37,20 @@ const getFlagged = async (request, response) => {
 
 const getInactive = async (request, response) => {
   try {
+    console.log("Entered with ID: ", request.params);
```
Comment (Rizvida1) [inline API/admin/get.js]: Clear log

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

File: API/user/get.js (hunk 1 above)
Comment (SukhsimranpreetSekhon) [reply API/user/get.js]: Not used at all. It was made for testing purposes, I forgot to delete it.

File: API/user/post.js (hunk 2 above)
Comment (SukhsimranpreetSekhon) [reply API/user/post.js]: Fixed.

File: test/user/get.test.js (hunk 3 above)
Comment (SukhsimranpreetSekhon) [reply test/user/get.test.js]: Fixed.

File: test/user/get.test.js (hunk 4 above)
Comment (SukhsimranpreetSekhon) [reply test/user/get.test.js]: Fixed.

File: API/admin/get.js (hunk 5 above)
Comment (SukhsimranpreetSekhon) [reply API/admin/get.js]: Not needed, was for testing purposes that I forgot to delete.

File: API/admin/get.js (hunk 6 above)
Comment (SukhsimranpreetSekhon) [reply API/admin/get.js]: Fixed.

**Review by Rizvida1 — APPROVED** [review]

**Pull request merged**